src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToUseAnalyzers.cs

```diff
@@ -75,7 +75,24 @@ private void VerifyRequestDelegateGeneratorIsUsed(TestAsset asset, bool? expectE
 
             var analyzers = command.GetValues();
 
-            Assert.Equal(expectEnabled ?? false, analyzers.Any(analyzer => analyzer.Contains("Microsoft.AspNetCore.Http.RequestDelegateGenerator.dll")));
+            Assert.Equal(expectEnabled ?? false, analyzers.Any(analyzer => analyzer.Contains(dllFileName)));
```

Comment:
This test won't work until you have the new ASP.NET Targeting pack with the generator added, however you probably want the functionality in place in the SDK targets before that (so that folks don't get it on-by default). 

Perhaps you can add a workaround temporarily to the test case that includes a fake analyzer item, that the target can remove / leave behind.

Reply:
👍 makes sense.